modules/caddytls/internalissuer.go

```diff
@@ -68,18 +67,20 @@ func (InternalIssuer) CaddyModule() caddy.ModuleInfo {
 func (iss *InternalIssuer) Provision(ctx caddy.Context) error {
 	iss.logger = ctx.Logger(iss)
 
+	// set some defaults
+	if iss.CA == "" {
+		iss.CA = caddypki.DefaultCAID
+	}
+
```

Comment:
Not sure why this moved to above L76-79, but I guess that's ok. :man_shrugging:

Reply:
I moved it to match the code in `acmeserver.go` which is very similar